test/e2e/kubelet_perf.go

```diff
@@ -73,16 +73,25 @@ var _ = Describe("Kubelet", func() {
 	})
 
 	Describe("resource usage tracking", func() {
-		density := []int{0, 50}
-		for _, podsPerNode := range density {
+		type trackingTests struct {
+			podsPerNode  int
+			onlyRunOnGCE bool
+		}
+		tests := []trackingTests{
+			{podsPerNode: 0, onlyRunOnGCE: false},
+			// 50 is greater the the default --max-pods on kubelet; only run it
+			// on GCE.
+			{podsPerNode: 50, onlyRunOnGCE: true},
```

Comment:
Why we want to skip GKE here? Shouldn't this is a good test on whatever Jenkin project which is designed to run slow tests?